### ISSUE TITLE
Preferences/Sync

### ISSUE THREAD

**Issue by rapisho** [louischan/simplewhite#60]
In the preferences the sync page isnt showing any Text, like its invisible.

**Comment by 3marcusw** [comment]
does the same bug appear in [Simple White X](https://addons.mozilla.org/en-us/firefox/addon/simplewhitex/?src=cb-dl-created), the up to date fork or [Simple White Compact](https://addons.mozilla.org/en-US/firefox/addon/simple-white-compact/), the compact fork?

**Comment by rapisho** [comment]
No, in both I can see text on Sync page and also the preferences are styled. Thanks.
